python/Ganga/new_tests/GPI/Bugs/Issue185.py

```diff
@@ -0,0 +1,31 @@
+from ..GangaUnitTest import GangaUnitTest
+
+class Issue185(GangaUnitTest):
```

Comment:
Where is the place to put this to make sure it's run then? It certainly doesn't need the GPI but from what I could glean from Jenkins, we're only running tests in `new_tests/GPI`...